src/js/modules/GET-breeds-home.js

```diff
@@ -0,0 +1,14 @@
+const ApiBreedsUrl = ('https://api.TheDogAPI.com/v1/breeds');
+
+const getBreeds = () => { // eslint-disable-line
+  return fetch(ApiBreedsUrl, {
+    method: 'GET',
+    headers: {
+      'x-api-key': 'be1ccef0-f839-40e0-8774-5ff5eeb18b34',
+    },
+  })
+    .then((response) => response.json())
+    .then((data) => data);
+};
```

Comment:
Please, try not to use linters disablers. This function immediately returns, so it can be written as
```JavaScript
const getBreeds = () => fetch(ApiBreedsUrl, {
...
});
```
which won't trigger a linter error. Another option is to use async instead of chaining `.then` in the promise.

Reply:
Ok Lucas, thank you, I will try find the way to run it with the linters, in the last project I have some issues calling the API without the return, I will improve that :smile: .